Mastodon/Scene/HashtagTimeline/HashtagTimelineHeaderView.swift

```diff
@@ -0,0 +1,181 @@
+//
+//  HashtagTimelineHeaderView.swift
+//  Mastodon
+//
+//  Created by Marcus Kida on 22.11.22.
+//
+
+import UIKit
+import CoreDataStack
+import MastodonSDK
+import MastodonUI
+import MastodonAsset
+import MastodonLocalization
+
+fileprivate extension CGFloat {
+    static let padding: CGFloat = 16
+    static let descriptionLabelSpacing: CGFloat = 12
+}
+
+final class HashtagTimelineHeaderView: UIView {
+    struct Data {
+        let name: String
+        let following: Bool
+        let postCount: Int
+        let participantsCount: Int
+        let postsTodayCount: Int
+        
+        static func from(_ entity: Mastodon.Entity.Tag) -> Self {
+            Data(
+                name: entity.name,
+                following: entity.following == true,
+                postCount: (entity.history ?? []).reduce(0) { res, acc in
+                    res + (Int(acc.uses) ?? 0)
+                },
+                participantsCount: (entity.history ?? []).reduce(0) { res, acc in
+                    res + (Int(acc.accounts) ?? 0)
+                },
+                postsTodayCount: Int(entity.history?.first?.uses ?? "0") ?? 0
+            )
+        }
+        
+        static func from(_ entity: Tag) -> Self {
+            Data(
+                name: entity.name,
+                following: entity.following,
+                postCount: entity.histories.reduce(0) { res, acc in
+                    res + (Int(acc.uses) ?? 0)
+                },
+                participantsCount: entity.histories.reduce(0) { res, acc in
+                    res + (Int(acc.accounts) ?? 0)
+                },
+                postsTodayCount: Int(entity.histories.first?.uses ?? "0") ?? 0
+            )
+        }
+    }
+    
+    let titleLabel = UILabel()
+
+    let postCountLabel = UILabel()
+    let participantsLabel = UILabel()
+    let postsTodayLabel = UILabel()
```

Comment:
The alignment count/type feels kind of weird? Why not center all?

<img width="207" alt="Screenshot 2022-11-30 at 18 04 55" src="https://user-images.githubusercontent.com/2580019/204862036-a645d7c3-01ff-4b21-b6e6-561db97c84bf.png">

Reply:
You're right 😆 somehow I didn't notice this, will fix it!

Reply:
OMG the more I look at this the worse it gets 🥹